tox.ini

```diff
@@ -109,7 +109,7 @@ commands =
 changedir = {toxinidir}
 deps =
     {env:LSR_ANSIBLE_DEP:ansible}
-    docker
+    docker{env:LSR_MOLECULE_DOCKER_VERSION:}
```

Comment:
Works for me as a workaround. Since there are plans for podman support, it might be nicer to use something like `LSR_MOLECULE_CONTAINER_DEPENDENCY` in the future and have runtox set it to `podman`, `docker` or `dockere<4.3`.